advice/emilyadvice.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>SkillCrush advice</title>
+        <link href="/Users/apple/Desktop/105-project-master/105-practice/css/emilyadvice.css" rel="stylesheet" type="text/css">
```

Comment:
Hi! Here you are using an _absolute_ link - meaning that it is the full path to your file on your computer. What we want to do here instead is use a _relative_ link - so that the CSS can connect relative to this file rather than a specific location on your computer - which makes it flexible and portable to other environments / servers. To step back one directory, you would use  `..` and then to move into the css directory, you would use `/css` and then finally the name of your stylesheet - making the full relative path: `../css/emilyadvice.css`